core/src/main/resources/hudson/PluginManager/_table.css

```diff
@@ -10,7 +11,8 @@ time {
 
 #filter-container {
     margin: 1em;
-    text-align: right;
+    text-align: center;
+    font-size: 2em;
```

Comment:
The search input doesn't seem to fit with the rest of the page to me, did it's size need to be increased?

Reply:
I did these changes to make it more noticeable. Now there's no way to use the "Available" tab without it, after all. Also larger font size for the placeholder list item.

Whether these need to apply to other tabs too is more questionable.

Reply:
alternative design:
https://github.com/daniel-beck/jenkins/pull/2

Reply:
@fqueiruga PTAL

Reply:
I'd just go with Tim's design, it's more consistent with the header one. We could also extract it into a searchbox widget in the future.